sql/moz-fx-data-shared-prod/fenix_derived/clients_yearly_v1/metadata.yaml

```diff
@@ -19,7 +20,6 @@ bigquery:
     type: day
     field: submission_date
     require_partition_filter: true
-    expiration_days: null
```

Comment:
should this have expiration set?

Reply:
This table has been added to the exclusion list.